### ISSUE TITLE
Liste FR blocks AG Pro lists update

### ISSUE THREAD

**Issue by Quorum75** [AdguardTeam/AdguardForiOS#2091]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

- **AdGuard version:**
  -  AG Pro 4.4.4 (965)
- **Device model and storage size:**
  -  iPhone 13 Pro Max 
- **Operating system and version:**
  -  iOS 16.2
- **Browser or App:**
  - 

### Expected Behavior

 Being able to update AG list FR

### Actual Behavior

When I manually update iOS AdGuard Pro I see in my NextDNS log that Liste FR is blocking filters.adtidy.org

→ 1278313086.rsc.cdn77.org | Blocked by Liste FR.

What's happening here?

### Screenshots

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/49731083/208242348-4a6f3f73-2b16-4b8a-9f70-8a36ca483062.png)

</details>

**Comment by Versty** [comment]
@Quorum75 Hi! This issue is not reproduces on our side. 
Does it still persist and reproduce on your device with the latest version of AdGuard Pro (4.4.5)?

**Comment by dnmTX** [comment]
@Versty what do you mean is not reproducible????
NextDNS simiriarly like AdGuard blocks domains from the pointed user lists as well as (their) CNAME's respectably.
When we take into account here that `1278313086.rsc.cdn77.org` is a CNAME of `filters.adtidy.org` and is listed in that **Liste FR** for some reason and because of this both domains will be blocked by NextDNS and AdGuard will not be able to update it's filters.
Bottom line is that this indeed is not AdGuard issue and @Quorum75 has to address it with the mainteners of that list(unless the mentioned list is one that AG offers and maintains). But to say that is not **reproducible**,is the wrong response here!

**Comment by Quorum75** [comment]
In fact I did report it to the maintener. 

https://github.com/easylist/listefr/issues/109

**Comment by Versty** [comment]
@dnmTX @Quorum75 
Liste FR contains no such rule `1278313086.rsc.cdn77.org`. The rules in this list were created to block elements using a special filtering syntax and are not suitable for DNS filtering. 
In the Liste FR we have `||cdn77.org^$popup, third-party`, which works for DNS filtering as `||cdn77.org^` because of  the difference in syntax.  

You can read more about [ad blocking syntax](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters) and [DNS filtering rules syntax](https://adguard-dns.io/kb/general/dns-filtering-syntax/) on this two pages.

**Comment by dnmTX** [comment]
@Quorum75 just whitelist `cdn77.org` on your end and be done with it. Obviously whoever converts those adblock filters into DNS filters needs to pay more attention as domains that are listed with additional filtering sintax(`third-party`...etc) should not be included at all,but instead they made their scripts to remove only the sintax part.
